stubs/src/radio/radio-controller.ts

```diff
@@ -1,63 +1,58 @@
 import * as dgram from 'dgram';
-import {UdpResponse} from './dataflow';
+import {UdpResponse, UDPScanConfig, UDPScanResults} from './dataflow';
 
 /**
  * A default radio timeout, in milliseconds.
  */
 const RADIO_TIMEOUT = 2500;
-
 /**
- * A class to contain all parameters required to perform
- * a UDP scan.
+ * A helper function to create timeout promises.
+ * @returns  A new Promise that resolves after RADIO_TIMEOUT milliseconds.
```

Comment:
This doesn't resolve after `RADIO_TIMEOUT` but after a parameter.

Reply:
Fixed.